src/index.js

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import hubsConfig from "./hubs.yaml";
+import MarkdownIt from "markdown-it";
+import { render } from "react-dom";
+import { Center, Image, RadioGroup, Radio, Stack, ChakraProvider, Container, Heading, Text, Link, Box, Flex, Button, baseStyle } from "@chakra-ui/react";
+import rstudioLogo from "./logos/rstudio.svg";
+import jupyterLogo from "./logos/jupyter.svg";
+
+const md = MarkdownIt();
+const pageProperties = window.pageProperties;
+
+const FAQItem = ({ q, a, ...props }) => {
+  return <Box key={q} {...props}>
+    <Heading
+      fontSize="lg" marginBottom={2}
+      dangerouslySetInnerHTML={{ __html: md.render(q) }} />
+    <Text dangerouslySetInnerHTML={{ __html: md.render(a) }} />
+  </Box>
+}
+
+const Welcome = ({ title, subTitle, ...props }) => {
+  return <Box {...props}>
+    <Heading
+      fontSize="xl" marginBottom={2}
+      dangerouslySetInnerHTML={{ __html: md.render(title) }} />
+    <Text fontSize="md" dangerouslySetInnerHTML={{ __html: md.render(subTitle) }} />
+  </Box>
+}
+
+const Interfaces = ({ interfaces, ...props }) => {
+  const interfaceDetails = {
+    rstudio: {
+      url: "https://rstudio.com",
+      logo: rstudioLogo,
+      width: 48
+    },
+    jupyter: {
+      url: "https://jupyter.org",
+      logo: jupyterLogo,
+      width: 56
+    }
+  }
+  return <Flex
+    flexDir="column" alignItems="end" justifyContent="center"
+    {...props}>
+    {interfaces.map(i => {
+      const iface = interfaceDetails[i];
+      return <Link href={iface.url} key={i} marginBottom={4}>
+        <Image width={iface.width} src={iface.logo} />
+      </Link>
+    })}
+  </Flex >
+}
+
+const LoginPanel = ({ ...props }) => {
+  const urlParams = new URLSearchParams(window.location.search);
+  // If the user is being sent to a given URL after logging in (via ?next),
+  // we want to say 'Log in to Continue'. If they came to the hub directly,
+  // we want to say 'Log in to start'
+  const nextUrl = urlParams.get('next');
+  const freshPage = nextUrl === null || nextUrl === '' || nextUrl === '/hub/';
+  console.log(urlParams.get('next'))
+
+  return <Flex alignItems="center" direction="column" {...props}>
+    {freshPage &&
+      <RadioGroup defaultValue="notebook" paddingBottom={4}>
+        <Stack spacing={4} direction="row">
+          <Text>After logging in, open</Text>
+          <Radio value="notebook">Jupyter Notebook</Radio>
+          <Radio value="rstudio">RStudio</Radio>
```

Comment:
Any reason to have RStudio as the second option? I would expect Lab before RStudio unless you want to five R stuff more prominence...

Reply:
@damianavila ah, was just reproducing what's there now. Also breaks up the Jupyter monotony :D